products/xinterpdf.rst

```diff
@@ -0,0 +1,70 @@
+##########
+xINTERPDF
+##########
+
+xINTERPDF is a Python GUI program for analyzing X-ray pair distribution function (PDF) data of organic compounds collected at synchrotron and/or laboratory X-ray sources. It uses DiffPy-CMI as a backend for simulation of PDFs. Currently it supports (1) The study of intermolecular interaction (e.g. hydrogen bonds) by subtracting out the scattering signal(s) of single molecule(s) in real space. (2) The PDF model fit of the crystalline organic compound using the method proposed by Prill et al. (*J. Appl. Cryst.*, **48**, 171-178, 2015.) (3) The phase quantification of physical mixtures of organics. (4) Generate Score/Scree plots based on Principle Component Analysis (PCA).
```

Comment:
- consider changing to [numbered list](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks)
- convert *J. Appl. Cryst., ...* to a hyperlink to http://dx.doi.org/10.1107/S1600576714026454